plugins/linux-v4l2/v4l2-output.c

```diff
@@ -98,6 +99,12 @@ static int loopback_module_load()
 	return run_command(
 		"pkexec modprobe v4l2loopback exclusive_caps=1 card_label='OBS Virtual Camera' && sleep 0.5");
 }
+#else
+bool loopback_module_available()
+{
+	return true;
```

Comment:
The goal of this is to not display the virtual webcam option unless the users system is correctly configured, what does correctly configuring a freeBSD platform look like? Based on your description it at a minimum requires running some sort of user daemon.

Reply:
Hi,

In FreeBSD the V4L2 loopback device driver is running in user-space via CUSE. It is not so easy to detects its presence, except trying to scan all the /dev/video<N> devices for the loopback one. Is this an acceptable solution for loopback_module_available() ?

--HPS

Reply:
If obs  cannot start the driver itself like on linux that seems good.